packages/perseus/src/components/sortable.tsx

```diff
@@ -771,35 +744,51 @@ class Sortable extends React.Component<SortableProps, SortableState> {
             return i.key === item.key;
         });
 
-        // @ts-expect-error [FEI-5003] - TS2551 - Property 'splice' does not exist on type 'readonly SortableItem[]'. Did you mean 'slice'?
         nextItems.splice(currentIndex, 1);
-        // @ts-expect-error [FEI-5003] - TS2551 - Property 'splice' does not exist on type 'readonly SortableItem[]'. Did you mean 'slice'?
         nextItems.splice(index, 0, item);
 
         this.setState({items: nextItems});
-        this.props.onChange && this.props.onChange({});
+        this.props.onChange({});
     }
 
-    onMouseMove(key: SortableItem["key"]) {
+    onMouseMove = (key: SortableItem["key"]) => {
         // Dragging: Rearrange items based on draggable's position
         // eslint-disable-next-line react/no-string-refs
-        // @ts-expect-error [FEI-5003] - TS2769 - No overload matches this call.
-        const $draggable = $(ReactDOM.findDOMNode(this.refs[key]));
-        // @ts-expect-error [FEI-5003] - TS2769 - No overload matches this call.
-        const $sortable = $(ReactDOM.findDOMNode(this));
-        const items = _.clone(this.state.items);
+        const draggableNode = ReactDOM.findDOMNode(this.refs[key]);
+        const sortableNode = ReactDOM.findDOMNode(this);
+
+        if (!draggableNode || !sortableNode) {
+            return;
+        }
+
+        const $draggable = $(draggableNode);
+        const $sortable = $(sortableNode);
+
+        const draggableOffset = $draggable.offset();
+        const sortableOffset = $sortable.offset();
+
+        if (!draggableOffset || !sortableOffset) {
+            return;
+        }
```

Comment:
I moved the calls to `$` and `.offset()` up so that we could do an early return if anything return `undefined`.  This shouldn't impact the semantics of this function.